lib/ProductOpener/Ingredients.pm

```diff
@@ -200,7 +200,7 @@ my %may_contain_regexps = (
 		"Dit product kan sporen van|bevat mogelijk sporen van|Kan sporen bevatten van|Kan sporen van|bevat mogelijk|sporen van",
 	nb =>
 		"kan inneholde spor av|kan forekomme spor av|kan inneholde spor|kan forekomme spor|kan inneholde|kan forekomme",
-	pl => "może zawierać śladowe ilości|produkt może zawierać|może zawierać|możliwa obecność",
+	pl => "może zawierać śladowe ilości|produkt może zawierać:?|może zawierać:?|możliwa obecność",
```

Comment:
why did you add :? shouldn't it be just ? (if you want to make that word optional)

Reply:
I wanted to make ':' optional